packages/linter/src/rules/RuntimeIsPreloaded.ts

```diff
@@ -3,14 +3,13 @@ import { Rule } from "../rule";
 
 export class RuntimeIsPreloaded extends Rule {
   run({ $ }: Context) {
-    const attr = [
-      "href='https://cdn.ampproject.org/v0.js'",
-      "rel='preload'",
-      "as='script'",
-    ]
+    const jsAttr = ["href$='/v0.js'", "rel='preload'", "as='script'"]
       .map((s) => `[${s}]`)
       .join("");
-    const isPreloaded = $(`link${attr}`).length > 0;
+    const mjsAttr = ["href$='/v0.mjs'", "rel='modulepreload'"]
```

Comment:
We're currently not using `modulepreload` so please omit checking for this attribute

Reply:
For mjs we now simply check if the rel attribute ends with preload, so normal preload (as it is used now) or modulepreload will be fine.